test/test_nested.py

```diff
@@ -13,13 +13,18 @@ class Nested:
         x: dcs.I64
         y: Annotated[bytes, 3]
 
+    assert dcs.get_struct_size(Nested) == (8) + (3)
+
     @dcs.dataclass()
     class Container:
         x: dcs.I32
         item1: Annotated[Nested, dcs.PadBefore(10)]
         item2: Annotated[Nested, dcs.PadAfter(12)]
         y: dcs.I32
 
+    # TODO: Should this be assuming alignment size?
```

Comment:
You could use `struct.calcsize(<format str>)` to calculate the expected size instead, then any native alignment issues will be handled.